doc-content/enterprise-only/processes/interacting-with-processes-viewing-adding-comments-proc.adoc

```diff
@@ -6,9 +6,12 @@ You can add comments to a task and also view the existing comments of a task in
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Track* -> *Task Inbox*.
 . On the *Task Inbox* page, click the task to open it.
-. On the task page, click the *Comments* tab.
-. In the *Comment* field, enter the comment and click *Add Comment*.
+. On the task page, click the *Work* tab or the *Comments* tab.
+. In the *Comment* field, enter the task related comment and click *Add Comment* icon.
++
+All task related comments are displayed in a tabular form in the *Work* as well as *Comments* tab.
+
 [NOTE]
 ====
-All task comments are displayed in a tabular form in the *Comments* tab.
+To select or clear the *Show task comments at work tab* check box, go to the project’s home page, click the *Settings* icon and select the *Process Administration* option. Only users with the `process-admin` role have access to enable or disable this feature.
```

Comment:
_go to the project’s home page_, that would Business Central home page, not really related to a project in specific.

Reply:
`process-admin`, that would be 'admin' role.